src/shared-components/dropdown/test.tsx

```diff
@@ -104,7 +105,10 @@ describe('<DesktopDropdown />', () => {
         />,
       );
 
-      wrapper.find('#select-input-box').simulate('click');
+      wrapper
+        .find('#select-input-box')
```

Comment:
After adding the `css` prop to the selector `div`, this query selector began returning multiple nodes. Selecting the first one produced the correct behavior, though I'm not exactly sure why...

Reply:
If you add some snapshots (for the before and after) it might be easy to see the change in the diff.

Reply:
Additionally, since that looks like an ID, we might definitely want to check that because there should only be one node, right?

Reply:
Good call on the snapshot test - will add that in now. Yeah, I'm kind of confused as to what is happening. It seems like adding the `css` prop (regardless of the content) was causing this to return a second node...